src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -263,6 +264,11 @@
 					.examples("on tool break:")
 					.since("2.1.1");
 		}
+		Skript.registerEvent("Item Damage", SimpleEvent.class, PlayerItemDamageEvent.class, "(tool|item) damag(e|ing)")
```

Comment:
Wouldn't this also call for armor damage? If so, ``on tool damage`` triggering for armor damage also seems weird.

Reply:
As far as I know, it gets called for armor damage. We could drop the tool from syntax, or keep it and filter armor out before event gets passed to scripts. The latter is a lot of work for a feature that will likely not be used by many.

Reply:
You're right. Ill remove tool and leave it with just item.